docs/ros/tutorials/cheat_sheet.mdx

```diff
@@ -0,0 +1,495 @@
+---
+title: ROS Cheat Sheet
+sidebar_label: ROS Cheat Sheet
+sidebar_position: 1
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+:::note
+
+This page is intended to provide quick reference to common ROS commands and concepts. It is not
+an exhaustive resource.
+
+Please refer to [the official ROS docs](https://docs.ros.org/) for complete documentation.
+
+:::
+
+:::note
+
+In any `bash` command, items contained within square brackets (`[]`) means the items with the
+brackets are _optional_, and may be omitted.
+
+:::
+
+## ROS command-line tools
+
+Since ROS 2 launched, all ROS 2 terminal commands are of the format `ros2 command [verb] [args...]`
+
+### Seting the `ROS_DOMAIN_ID`
+
+Because of its distributed nature, ROS 2 uses numbered **domains** to manage communication
+between nodes. Before running any `ros2` commands, make sure your domain is set correctly.
+
+```bash
+export ROS_DOMAIN_ID=0
+```
+
+Domain IDs can be any integer from `0` to `232`, though general convention is to avoid using
+any domain ID higher than `100` unless absolutely necessary.
+
+If unset, the default domain ID is `0`.
+
+### Interacting with nodes
+
+#### Start a node
+
+```bash
+ros2 run package_name executable_name [args]
```

Comment:
Should we add the params using `--ros-args` and a namespace?

Reply:
Good idea. I've added that to the example.